src/main/java/kostka/moviecatalog/service/runtimeconfiguration/RuntimeConfigurationService.java

```diff
@@ -0,0 +1,65 @@
+package kostka.moviecatalog.service.runtimeconfiguration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import kostka.moviecatalog.dto.RuntimeConfigDto;
+import kostka.moviecatalog.entity.runtimeconfiguration.RuntimeConfiguration;
+import kostka.moviecatalog.enumeration.RuntimeConfigurationEnum;
+import kostka.moviecatalog.exception.MissingRuntimeConfigurationException;
+import kostka.moviecatalog.repository.RuntimeConfigRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+import static kostka.moviecatalog.factory.RuntimeConfigurationFactory.getOptionsClass;
+
+@Service
+public class RuntimeConfigurationService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RuntimeConfigurationService.class);
+    private RuntimeConfigRepository runtimeConfigRepository;
+
+    private ObjectMapper mapper = new ObjectMapper();
```

Comment:
Optional - ten ObjectMapper by se dal pouzivat jako singleton v cele aplikaci. Takze by si z toho mohl udelat beanu. Jde to udelat tak, ze tady das klasicky autowire v konstruktoru a beanu vytvoris v nejakem @Configuration souboru takto:
```java
@Bean
public ObjectMapper objectMapper() {
    return new ObjectMapper();
}
```

Spring ti u takove metody zaruci to, ze se zavola jen jednou a pak uz se bude ta beana pouzivat jako singleton.

Reply:
I am aware of it, thanks - maybe I will create separate issue for it